transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessagingClient.kt

```diff
@@ -294,6 +294,13 @@ interface MessagingClient {
     @Throws(IllegalStateException::class)
     fun logoutFromAuthenticatedSession()
 
+    /**
+     * Check if the user needs to be authorized.
+     *
+     * @param callback A callback function that will be called with a boolean indicating whether the user needs to be authorized or not.
```

Comment:
Consider this statement, as it is a little bit less cumbersome: 

@param callback A function that receives `true` if authorization is needed, or `false` otherwise.